lib/components/settings/update_manager.dart

```diff
@@ -190,13 +215,36 @@ abstract class UpdateManager {
 
     OpenFilex.open(file.path);
   }
+
+  static Future<String> getNewestVersionChangelog() async {
+    final int newestVersion;
+    newestVersion = await getNewestVersion() ?? 0;
```

Comment:
Please can you reuse the previous call of `getNewestVersion()` from `_checkForUpdate()` so it's not run twice?